docs/dev/testing.md

```diff
@@ -302,6 +302,57 @@ suite is as solid as possible. The first step is contributing a stable (read: no
 To do so, when developing tests, please run them (1) in debug mode and (2) 100 times in a row (using `--repeat 100`),
 and see that they pass successfully.
 
+## Allure reporting
+
+To make analyzing of the test results more convenient, an allure reporting tool is introduced.
+Python module allure-pytest module is included in the toolchain image and additional parameter added to gather allure
+data for the test run.
+However, the allure binary is not a part of the toolchain image.
+So to have the full benefit of the new reporting tool allure binary should be installed locally.
+
+### Allure installation
+
+To install allure tool please follow official documentation https://allurereport.org/docs/install-for-linux/
+
+**Note:** rpm package requires dependency `default-jre-headless` but on Fedora 38 none of the packages provides it.
+So manual installation of allure is required.
+
+#### Manual allure installation
+
+Allure provides the package with all dependencies in tgz archive.
+This option can be used on Ubuntu or Fedora if rmp or deb package cannot be used for any reason.
+
+The third party programs usually installed in the `/opt` directory on unix based systems.
+This is requiring the root permissions.
+Another location where allure can be placed is a `~/.local/share` directory that is not requiring the root permissions
+since it's a home folder of the current user.
+
+1. Download the tgz archive from the [release page](https://github.com/allure-framework/allure2/releases)
+2. Create destination directory where the allure binary will be located:\
+```shell
+$ sudo mkdir /opt/allure
+```
+3. Extract the tgz archive to the desired destination with a command:\
+```shell
+$ sudo tar -xf allure-2.29.0.tgz --directory /opt/allure --strip-components=1
+```
+4. Add allure binary to you PATH directly from `/opt` directory or add symlink to `~/.local/bin` which should be already in the PATH\
+```shell
+$ ln -s /opt/allure/bin/allure ~/.local/bin/allure
+```
```

Comment:
Kamil suggested dropping this part and I agree. Instructions on how to install allure, manually or not, are present on their website - and I suppose that they are maintaining those instructions. If you keep these instructions then we need to keep them in sync in case something changes there - it's an additional effort which I'm not sure is needed.

You could just provide a link to their documentation here instead - ofc it's also prone to breaking, but it's less effort maintaining just the link and easier to work around when it's not working.

I won't insist too strongly, but I think that dropping it would really be beneficial.